pgbot/commands/__init__.py

```diff
@@ -97,25 +97,26 @@ async def handle(invoke_msg: discord.Message, response_msg: discord.Message = No
     ):
         return
 
+    if response_msg is None:
+        response_msg = await embed_utils.send_2(
+            invoke_msg.channel,
+            title="Your command is being processed:",
+            fields=(("\u2800", "`Loading...`", False),),
+        )
+
     async with db.DiscordDB("bot_mutes") as db_obj:
         muted_users = db_obj.get({})
 
     if invoke_msg.author.id in muted_users:
-        await embed_utils.send_2(
+        response_msg = await embed_utils.send_2(
             invoke_msg.channel,
```

Comment:
These two lines
```py
response_msg = await embed_utils.send_2(
    invoke_msg.channel,
```
Should be replaced with
```py
await embed_utils.replace_2(
    response_msg,
```
Because we want to edit the message, not resend it